src/Humanizer/FrequencyHumanizeExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Humanizer
+{
+    public static class FrequencyHumanizeExtensions
+    {
+        public static string Humanize(this Frequency input)
+        {
+            string output = "every";
+            switch (input.OccurrenceFrequency)
+            {
+                case FrequencyEnum.Never:
+                    output = "never";
```

Comment:
I mean, to put these strings to .NET resources, so we can get translations out of the box